test/functional/write_concern.test.js

```diff
@@ -58,4 +61,40 @@ describe('Write Concern', function () {
       withMonitoredClient('insert', { queryOptions: { journal: true } }, journalOptionTest)
     );
   });
+
+  let server;
+  before(() => {
+    return mock.createServer().then(s => {
+      server = s;
+    });
+  });
+
+  after(() => {
+    mock.cleanup();
```

Comment:
This should also return (or just remove the curly braces)

Reply:
Ooo nice catch thanks!!